src/lib/utils/data-handlers/resources/resource.ts

```diff
@@ -237,10 +237,17 @@ export async function filterToAvailableAssociatedResourceContent(
     });
 }
 
-export function sortByDisplayName<T extends { displayName: string | undefined }>(resources: T[]): T[] {
-    return resources.sort((a, b) =>
-        (a.displayName ?? '').localeCompare(b.displayName ?? '', undefined, { numeric: true })
-    );
+export function sortByDisplayName<T extends { displayName: string | undefined; sortOrder?: number }>(
+    resources: T[]
+): T[] {
+    return resources.sort((a, b) => {
+        if (a.sortOrder !== undefined && b.sortOrder !== undefined) {
+            if (a.sortOrder !== b.sortOrder) {
```

Comment:
Does this need to be an additional `if` as opposed to just `&&` in the above?

Reply:
Probably not but I had set to auto-merge, I'll address in my next PR